src/parser.cpp

```diff
@@ -1,3 +1,4 @@
+#include <iostream>
```

Comment:
Do you need iostream?

Reply:
No, you're right. Removed it, and updated the benchmarks :-)